app/controllers/statuses_controller.rb

```diff
@@ -54,6 +54,7 @@ def embed
     skip_session!
     expires_in 180, public: true
     response.headers['X-Frame-Options'] = 'ALLOWALL'
+    @animate_custom_emoji = params[:animate_custom_emoji] == "1"
```

Comment:
Maybe name the param "autoplay" and use Rails truthiness parser / boolean type cast used in other parts of the code.

Make it control not only custom emoji but avatars and media too

Reply:
I did it, but it seems even with autoPlayGif set to true, the gifs do not autoplay in the public pages :thinking: